draft-ietf-ipsecme-ikev2-multiple-ke-00.xml

```diff
@@ -695,8 +703,8 @@ SKEYSEED(n) = prf(SK_d(n-1), SK(n) | Ni | Nr)
 
                 <t> where SK(n) is the resulting shared secret of this key exchange,
                 Ni and Nr are nonces from the IKE_SA_INIT exchange and SK_d(n-1) is the 
-                last generated SK_d, (derived from the previous IKE_INTERMEDIATE exchange, 
-                or the IKE_SA_INIT if there have not already been any IKE_INTERMEDIATE exchanges).
+                last generated SK_d, (derived IKE_SA_INIT for the first use of IKE_INTERMEDIATE,
+                otherwise from the previous IKE_INTERMEDIATE exchange).
```

Comment:
```suggestion
                last generated SK_d, (derived from IKE_SA_INIT for the first use of IKE_INTERMEDIATE,
                otherwise from the previous IKE_INTERMEDIATE exchange).
```
There seems to be a typo.